cookbook/smart-content-data-provider.rst

```diff
@@ -98,6 +98,28 @@ The rest of the functionality and Query generation is done in the Trait.
     Be sure that the returned entities has valid serialization configuration for
     JMS\Serializer.
 
+There are following hooks to influence the query generation. These are functions
+which are optional to override in the repository.
+
+.. list-table::
+    :header-rows: 1
+
+    * - Name
+      - Description
+    * - append(QueryBuilder $queryBuilder, $alias, $locale, $options = [])
+      - Additional select, where or joins can be added to the query to match
```

Comment:
put `select`, `where` and `joins` in backticks to separate them from the rest of the text.